forte/data/selector.py

```diff
@@ -39,7 +39,7 @@
 
 class Selector(Generic[InputPackType, OutputPackType]):
     def __init__(self, **kwargs):
-        pass
+        self._stored_kwargs = kwargs
```

Comment:
Looks like now all the `Selector` implementation need to add a call to `__init__` and pass in the values. I think this would create confusion. I think there are two ways:

1. Use `locals` (looks hacky)
https://stackoverflow.com/questions/3136915/passing-all-arguments-of-a-function-to-another-function/3137022#3137022
We can call `locals` in the first line of the base `__init__`, which should probably save only the function parameters

2. Add a `todict` and `fromdict` function in the `Selector` base instead, and implement the functions in the following way:
   a. `todict`, put all values in `dict` to JSON
   b. `fromdict`, load all values from this.

We will allow the users to implement `todict` and `fromdict` in their own way. If they don't, we will use the default implementation.